frontend/src/components/question-creation/QuestionElement.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { HStack, Box } from "@chakra-ui/react";
+import { DeleteOutlineIcon, HamburgerMenuIcon } from "../../assets/icons";
+
+export interface QuestionElementProps {
+  children: React.ReactNode;
+}
+
+const QuestionElement = ({
+  children,
+}: QuestionElementProps): React.ReactElement => {
+  return (
+    <HStack spacing="6">
+      <Box color="grey.300">
+        <HamburgerMenuIcon />
+      </Box>
+      {children}
+      <Box color="grey.300">
+        <DeleteOutlineIcon />
+      </Box>
```

Comment:
Is this the best way to add colour to an icon? I couldn't get it to work with the <Icon /> Chakra component for some reason.

Reply:
ya this works, other option is to define the colour directly in `icons.tsx` but then the colour will apply for all instances of that icon

Reply:
okay!